projects/js-packages/charts/src/components/pie-chart/pie-chart.module.scss

```diff
@@ -1,4 +1,5 @@
 .pie-chart {
 	display: flex;
 	flex-direction: column;
+	overflow: hidden;
```

Comment:
This seems possibly unintentional/unrelated too...

Reply:
This one I believe is intentional/useful, but other overflow management rules were not intentional and caused problems :)